app/views/gardens/index.html.erb

```diff
@@ -1,71 +1,112 @@
-            <div class="index-plant-list">
-              <% garden.plants.each do |plant| %>
-                <div class="plant" id="plant-<%= plant.id %>"
-                  <p><%= link_to plant.name, plant_path(plant) %></p>
+    <section class="garden-container">
+      <% @user.own_gardens.each do |garden| %>
+        <div id="garden-<%= garden.id %>">
+          <div class="card-group">
+            <div class="card">
+              <h2 class="garden-name"><%= garden.name %></h2>
+              <div class="index-plant-list">
+                <% garden.plants.each do |plant| %>
+                  <div class="plant" id="plant-<%= plant.id %>"
+                    <p><%= link_to plant.name, plant_path(plant) %></p>
+                  </div>
+                <% end %>
+              </div>
+              <div class="btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
+                <div class="btn-group mr-2" role="group" aria-label="First group">
+                  <button type="button" class="btn btn-secondary">
+                    <%= link_to 'View Your Garden', garden_path(garden), class: "btn"%>
+                  </button>
+                  <button type="button" class="btn btn-secondary">
+                    <%= link_to 'Add Caretaker for this Garden', invite_path(garden.id), class: "btn" %>
+                  </button>
                 </div>
-              <% end %>
-            </div>
-            <div class="btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
-              <div class="btn-group mr-2" role="group" aria-label="First group">
-                <button type="button" class="btn btn-secondary">
-                  <%= link_to 'View Your Garden', garden_path(garden), class: "btn"%>
-                </button>
-                <button type="button" class="btn btn-secondary">
-                  <%= link_to 'Add Caretaker for this Garden', invite_path(garden.id), class: "btn" %>
-                </button>
               </div>
             </div>
-          </div>
 
-          <% if @facade.has_weather?(garden) %>
-            <div class="card">
-              <div class="weather-card-<%= garden.id %>">
-                <h3>Weather in <%= garden.name %> (<%= garden.zip_code %>):</h3>
-                  <% @next_seven_days.each_with_index do |date, index| %>
-                    <h5 class="weather_day"><%= date %>, Chance of Rain: <%= @facade.weather(garden).chance_of_rain(index).round(0) %>%</h5>
-                  <% end %>
+            <% if @facade.has_weather?(garden) %>
+              <div class="card">
+                <div class="weather-card-<%= garden.id %>">
+                  <h3>Weather in <%= garden.name %> (<%= garden.zip_code %>):</h3>
+                    <% @next_seven_days.each_with_index do |date, index| %>
+                      <h5 class="weather_day"><%= date %>, Chance of Rain: <%= @facade.weather(garden).chance_of_rain(index).round(0) %>%</h5>
```

Comment:
Let's make sure to merge the Precip Type PR first. As it affects the Garden Index and Dashboard page.